packages/lib/src/vis/heatmap/HeatmapMesh.tsx

```diff
@@ -201,26 +141,27 @@ function HeatmapMesh(props: Props) {
     fragmentShader: `
       uniform sampler2D data;
       uniform sampler2D colorMap;
-      uniform int scaleType;
-      uniform float alphaMin;
+
+      uniform float min;
+      uniform float oneOverRange;
+      uniform float gammaExponent;
+
       uniform sampler2D alpha;
+      uniform float alphaMin;
       uniform float oneOverAlphaRange;
       uniform int withAlpha;
 
       const vec4 nanColor = vec4(255, 255, 255, 1);
+      const float oneOverLog10 = 0.43429448190325176;
 
       varying vec2 coords;
 
-      ${
-        SCALE_SHADER[isScaleType(scaleType) ? scaleType : ScaleType.Gamma]
-          .fragment
-      }
+      ${SCALE_SHADER[isScaleType(scaleType) ? scaleType : ScaleType.Gamma]}
 
       void main() {
         float value = texture2D(data, coords).r;
-        bool isNotFinite = isnan(value) || isinf(value);
```

Comment:
I find `isNotFinite` quite confusing, so I just inlined the conditions. 😄

Reply:
Good idea, it is indeed clearer